modules/service/src/main/resources/db/migration/V0953__program_active_period.sql

```diff
@@ -0,0 +1,71 @@
+-- Add active start / end.  The defaults seem fairly random but correspond to
+-- the limiting dates for our LST calculation.
+
+ALTER TABLE t_program
+  ADD COLUMN c_active_start date NOT NULL DEFAULT '1901-01-01'::date,
+  ADD COLUMN c_active_end   date NOT NULL DEFAULT '2099-12-31'::date,
```

Comment:
What happens if the ODB makes it to the next century?

Reply:
There's definitely a Y2100 problem here.  But it won't be our problem.